src/Makefile

```diff
@@ -0,0 +1,22 @@
+# $FreeBSD$
+
+MAN=
+CFLAGS=	-Wall -lmd -lz -lfetch -I..
```

Comment:
OpenBSD isn't going to be happy with `-I..` but `-I${.CURDIR}/..` will work.

Reply:
This was previously ${.CURDIR}/.., but something happened. =) I'll fix this in another iteration, which will likely also do a s/common.h/ogit.h/ style rename.